ext/vulnmdsrc/nvd/nvd.go

```diff
@@ -165,46 +165,64 @@ func getDataFeeds(dataFeedHashes map[string]string, localPath string) (map[strin
 				}
 			}
 
-			// Download data feed.
-			r, err := http.Get(fmt.Sprintf(dataFeedURL, dataFeedName))
-			if err != nil {
-				log.WithError(err).WithField(logDataFeedName, dataFeedName).Error("could not download NVD data feed")
-				return dataFeedReaders, dataFeedHashes, commonerr.ErrCouldNotDownload
-			}
+			downloadAndSave := func() error {
```

Comment:
I think you should pull this out into a top-level function called downloadFeed, rather than an inline function. After that LGTM.